deploy-manage/deploy/elastic-cloud/differences-from-other-elasticsearch-offerings.md

```diff
@@ -240,6 +240,15 @@ When attempting to use an unavailable index setting, you'll receive this error:
 }
 ```
 
+## Elastic Cloud email service limits [cloud-email-service-limits]
+
+The following quotas apply to both {{ech}} deployments and {{serverless-full}} projects when using the Elastic email service:
+
+* Email sending quota: 500 emails per 15min period.
+* Maximum number of recipients per message: 30 recipients per email (To, CC and BCC all count as recipients).
```

Comment:
```suggestion
* Maximum number of recipients per message: 30 recipients per email (To, CC, and BCC all count as recipients).
```

Reply:
I've fixed that, sorry I had already moved the content somewhere else and I cannot commit that now.